### PR TITLE
Improve on the Swagger doc

### PR DESCRIPTION
**Pull request by ateska**
* change in `build_swagger_documentation` so that it propagates a host into the list of servers; it is more sane way of how to call "self" from Swagger, removing that relative URL
* improved way of how swagger is loaded to web server (thru index.html)
* added "shortcut" param "api" to `asab.web.create_web_server()` that enables swagger doc just by stating `api=True`

### CODE REVIEW

**Comment by mejroslav** [comment]
@ateska Can you summarize what this is about and if there are gonna be some TODOs?

**Comment by ateska** [comment]
Summarised.

**Comment by ateska** [comment]
Flake8 is broken, no other TODOs on the list.
Maybe re-alignment with the recent Swagger changes will be needed.

**Comment by ateska** [comment]
Flake8 is working now but the unit test in Python 3.9 is failing.

**Comment by mejroslav** [comment]
> Flake8 is working now but the unit test in Python 3.9 is failing.

That is probably ok. The unit test was designed to expect a different output from this function. I disabled it in master.

**Comment by mejroslav** [comment]
OK, I am taking this one.
